docs/DeveloperGuide.md

```diff
@@ -830,5 +830,5 @@ This is an example of the result of the enhancement:
     After the change, this will be a valid command: `add i/S5087089H n/John Doe p/98765432 a/John street, block 123, #01-01 c/7`.
 6. **Allow users to delete a call from the call history:** Currently, users can only add to the call history, using the `mark` command. We plan to allow users to delete a call from the call history. This is useful for users who have made a mistake in marking a call or have marked a call wrongly. The user can delete the call and mark the call again with the correct information. <br/><br/>
 
-    How we plan to implement this is to allow users to use the `delete` command in the history view. The user can specify the index of the call to delete. For example, `delete 1` will delete the first call in the call history. The `delete` command will remain unchanged, if the user is on the person list view.
+    How we plan to implement this is to allow users to use the `delete` command in the history view. The user can specify the index of the call to delete. For example, `delete 1` will delete the first call in the call history. Deleting by `NRIC` (e.g. `delete S1021013E`) will also be disabled in the call history view. The `delete` command will remain unchanged, if the user is on the person list view. 
```

Comment:
Does delete the NRIC shown in profile view in history list allowed? like user want to delete the person since we are going to enable other commands that use NRIC for the specified person shown in history list view.

Reply:
That's true, might overlap with enhancement 3